packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "kube_logs_tab_title": "Audit Logs",
+  "kube_logs_list_no_logs": "En attente de logs",
+  "kube_logs_description": "Vous trouverez ici les logs du control-plane de votre cluster Kubernetes. Pour observer vos pods métier, nous vous invitons à suivre la documentation.",
+  "kube_logs_description_link": "En savoir plus.",
+  "kube_logs_add_success": "Les limitations APIServer ont été modifiées avec succès.",
```

Comment:
Wrong copy/paste ?

Reply:
Yes right, i forgot to remove the rest.
Thanks.